functions/PushNotificationTutorConfirmation.js

```diff
@@ -0,0 +1,111 @@
+const {onDocumentUpdated} = require("firebase-functions/v2/firestore");
+const {setGlobalOptions} = require("firebase-functions/v2");
+const {initializeApp} = require("firebase-admin/app");
+const {getFirestore} = require("firebase-admin/firestore");
+const admin = require("firebase-admin");
+
+initializeApp();
+
+setGlobalOptions({
```

Comment:
Setting maxInstances, timeoutSeconds, and memory in setGlobalOptions improves function scalability and control as the number of notifications will increase and we might enounter problems with our Baze contract on firebase and we want to control the cost associate to the notifications